<?xml version="1.0" encoding="UTF-8"?>
<!--Author: Chris Lawson-->
<!--Contact: clawso21@une.edu.au-->
<!--Source: U-Infuse-->
<!--Purpose: Training Configuration Dialogue 1 UI-->
<ui version="4.0">
 <class>ModTrain</class>
 <widget class="QDialog" name="ModTrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Model Datasets and Classes</string>
  </property>
  <widget class="QLabel" name="classesLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Classes</string>
   </property>
  </widget>
  <widget class="QLabel" name="datasetsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dataset Folders</string>
   </property>
  </widget>
  <widget class="QListWidget" name="foldersList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="classesList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="yesButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Yes</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="negativeCheckBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>190</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Include Negative Images  </string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="negativeFoldersList">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>271</width>
     <height>161</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="negativesLabel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Negatives</string>
   </property>
  </widget>
  <widget class="QLabel" name="generateLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Training Files</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
